FoodTracker/Controllers/MealViewController.swift

```diff
@@ -16,7 +16,7 @@ class MealViewController: UIViewController {
     @IBOutlet weak var saveButton: UIBarButtonItem!
     
     //MARK: Properties
-    var meal: Meal?
+    private var meal: Meal?
```

Comment:
Have you build it after changing it to private ? 
It seems no to me as this meal is being passed from other class so it should accessible from outside.
Always build and run the app after updating anything in the code for PR to verify that everything works as expected.